torax/interpolated_param.py

```diff
@@ -392,3 +397,10 @@ def get_value(self, x: chex.Numeric) -> chex.Array:
 TimeInterpolatedInput = (
     InterpolatedVarSingleAxisInput | tuple[InterpolatedVarSingleAxisInput, str]
 )
+# Type-alias for a variable to be interpolated in time and rho_norm.
```

Comment:
With this change, I think I'm happy with the naming we were discussing on the other commit!